lib/web_of_science.rb

```diff
@@ -22,5 +22,8 @@ def self.queries
     @@queries ||= WebOfScience::Queries.new(client)
   end
 
+  def self.logger
+    @@logger ||= Logger.new(Settings.WOS.LOG)
+  end
```

Comment:
This commit combines moving the namespace plus adding a new logger - these should be separate commits.  Split this commit into separate commits and then squash the addition of the logger with another commit that removes it from the NotificationManager, that commit is then a clear, atomic move-method refactor.

Reply:
Why nitpick commit composition for changes you ostensibly want?  We are literally never going to use either state independently.  For this PR (and the previous one I already extracted these from), the **point** of the rename is to provide the logger method at the top `WebOfScience` module.  Breaking them apart gains nothing.

Reply:
I notice that renaming the module and moving the logger method are currently in two commits, but each commit contains a bit of both.  Either squash them all or separate the move module commit from the move method commit.

Reply:
Yea, so just squash the first and last commits in this PR.

Reply:
Squashed.